### PR TITLE
Update stl_file to threemf_file for from_3mf

### PR DESCRIPTION
**Pull request by kiuber**

### CODE REVIEW

**Review by gyscos — APPROVED** [review]

**Comment by gyscos** [comment]
Might be worth renaming it also in `Mesh::load` just above.

File: src/mesh.rs [hunk 1]
```diff
@@ -101,49 +101,49 @@ pub struct Mesh {
...
+        match model_filename {
             "-" => {
                 // create_stl_reader requires Seek, so we must read the entire stream into memory before proceeding.
                 // So I guess this can just consume all RAM if it gets bad input. Hmmm....
-                let mut input_buffer = Vec::new();
-                io::stdin().read_to_end(&mut input_buffer)?;
-                Mesh::from_stl(Cursor::new(input_buffer), recalc_normals)
+                let mut model_buffer = Vec::new();
+                io::stdin().read_to_end(&mut model_buffer)?;
+                Mesh::from_stl(Cursor::new(model_buffer), recalc_normals)
```
Comment (gyscos) [inline src/mesh.rs:118]: Here `input_buffer` was fine

**Review by gyscos — APPROVED** [review]

File: src/mesh.rs (hunk 1 above)
Comment (unlimitedbacon) [reply src/mesh.rs:118]: Agreed. `input_buffer` is better here but the rest of this looks good. Make that change and I will merge.

Comment (kiuber) [reply src/mesh.rs]: done. thanks.

**Pull request merged**